ios/FluentUI/Core/Theme/Tokens/ShadowInfo.swift

```diff
@@ -27,11 +28,11 @@ public struct ShadowInfo: Equatable {
                 xTwo: CGFloat,
                 yTwo: CGFloat) {
         self.colorOne = colorOne
-        self.blurOne = blurOne
+        self.blurOne = blurOne / blurDivisor
         self.xOne = xOne
         self.yOne = yOne
         self.colorTwo = colorTwo
-        self.blurTwo = blurTwo
+        self.blurTwo = blurTwo / blurDivisor
```

Comment:
can we just make sure our global token is already have the right apple value rather than doing a divison?  @lyzhan7 what did you do for furn?

Reply:
We had a team discussion about this some time ago and agreed to use the figma values and just adjust them.

Reply:
We make the adjustment inside FURN as well, rather than in the token pipeline. I think we considered updating the tokens values but decided against it, @mischreiber @Saadnajmi might remember more about this